fairlearn/utils/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
```

Comment:
```suggestion
# Copyright (c) Fairlearn contributors.
```

maybe? didn't we change it to this?

Reply:
According to Miro's comment [here](https://github.com/fairlearn/fairlearn/pull/823#issuecomment-857661650)  removing existing copyright from the source code might be in conflict with the MIT license.

Reply:
Not if all contributors agree + this is new code with be license

Reply:
Good points, in that case I'm very happy to change it to Fairlearn contributors :)